### DISCUSSION TITLE
Why no hybrid_search method in class MilvusClient ?

### DISCUSSION THREAD

**Discussion by baiyuxiong** [milvus-io/milvus#35422] [Q&A and General discussion]
Do any one know how to do a hybrid_search by MilvusClient?

**Comment by yhmo** [comment]
This interface is not implemented in MilvusClient. Currently only the Collection class has this interface. I think this is a mistake, will add it in the next minor version.

**Reply by xiaofan-luan** [reply]
> there is a plan to support hybrid search in Milvus Client, this could be done by the end of august
